rest/data/interface.go

```diff
@@ -217,6 +217,7 @@ type Connector interface {
 	AddPublicKey(*user.DBUser, string, string) error
 	DeletePublicKey(*user.DBUser, string) error
 	UpdateSettings(*user.DBUser, user.UserSettings) error
+	SubmitFeedback(model.FeedbackSubmission) error
```

Comment:
can we have this type be the rest/model type rather than the db model type?

Reply:
Changed